eclipse-collections/src/main/java/org/eclipse/collections/impl/list/Interval.java

```diff
@@ -1063,7 +1063,7 @@ public Interval take(int count)
         {
             return Interval.fromToBy(this.from, this.locationAfterN(count - 1), this.step);
         }
-        return Interval.fromToBy(this.from, this.from, this.step);
```

Comment:
I think you should clarify in the release notes that the old behavior was to return an interval of size one. And move the note to the top because it's the first behavior change.